Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -116,7 +139,22 @@ def test_cohesity_restore_from_last_snapshot(self):
         )
         self.assertEqual(returncode, 0)
 
-        time.sleep(30)  # Sleep for 30 seconds
+        while True:
```

Comment:
Please move all the poll logic code including  asserting returncode inside run_playbook(). This is duplicated again in test_cohesity_close_helios_incident.

Reply:
done